content/blog/2021/containerqueries.md

```diff
@@ -63,34 +63,54 @@ container.
 ```scss
 article,
 section {
-  contain: layout size style;
+  container-type: inline-size;
 }
 ```
 
-The `contain` property is part of the existing [CSS Containment
-Module](https://drafts.csswg.org/css-contain/). The `layout` value activates
-[layout
-containment](https://drafts.csswg.org/css-contain/#valdef-contain-layout) on the
-container, which ensures that "nothing outside can affect its internal layout,
-and vice versa." `Size` also [currently
-exists](https://drafts.csswg.org/css-contain/#size-containment) as a value for
-the `contain` property, and enables laying out a containment box without
-accounting for its descendants in either the inline or block axes.
-`Style` [containment](https://drafts.csswg.org/css-contain/#style-containment)
-isolates counters that might exist inside of the container.
+The `container-type`^[_Syntax for establishing a containment context was updated
+**July 12, 2021** to use the new `container-type` and `container-name`
+properties instead of `contain`._] property turns on containment for the
+selected elements, allowing for the testing of aspects such as style, size, and
+state.
+
+The `inline-size` value specifies that, in this case, queries will be made
+against the containing element's [inline
+axis](https://drafts.csswg.org/css-writing-modes-4/#inline-axis).
+[`Layout`](https://drafts.csswg.org/css-contain/#valdef-contain-layout) and
+[`size`](https://drafts.csswg.org/css-contain/#size-containment)^[_`Style` added
+as necessary value for `contain` property when establishing a containment
+context._ Updated **May 07, 2021**.] containment are applied automatically as
```

Comment:
I don't really understand this second footnote. What was updated on May 7? It looks like `style` is now being removed in this update?

Reply:
@jgerigmeyer 

`size` was added as a required value of the old `contain` property on May 7. 

`contain: inline-size layout style;`

both `layout` and `style` are implied with the new `container-type` and `container-name` values, which replace `contain` in this context.

Tried to make that more clear:
![image](https://user-images.githubusercontent.com/41588129/125638354-6657c9a5-e8f2-47fd-8063-cb8448e4a7a7.png)